graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -173,7 +173,15 @@ def _get_fields(ast):
             # If we ever allow repeated field names,
             # then we have to change the Location naming scheme to reflect the repetitions
             # and disambiguate between Recurse and Traverse visits to a Location.
-            raise GraphQLCompilationError(u'Encountered repeated field name: {}'.format(name))
+            raise GraphQLCompilationError(u'Encountered repeated field name: {}. If you are '
+                                          u'attempting to specify multiple directives on a single '
+                                          u'property field, one way to do so is to follow the '
+                                          u'format: propertyField directive1 directive2 ...'
+                                          u'However, note that all whitespace sequences, '
+                                          u'(e.g. spaces, tabs and newline characters),  are '
+                                          u'treated in the same way as delimiters in GraphQL, '
+                                          u'so feel free to space out directives in a different '
+                                          u'fashion.'.format(name))
```

Comment:
This is way too technical, just to explain that if the line is too long they can break it up.